### PR TITLE
add scripts to check the consistency of users to projects

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Add scripts to check the consistency of users to projects. Closes https://github.com/CenterForOpenScience/osf.io/issues/1255

### CODE REVIEW

**Comment by chennan47** [comment]
@sloria rdy for review.

**Comment by lyndsysimon** [comment]
First pass complete - no issues found. :star:

File: scripts/consistency/consistency_of_users_to_projects.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                count += 1
+
+    if count == 0:
+        logger.info("Consistency of users to projects' check is done. There is no inconsistency found.")
+    else:
+        logger.info("Consistency of users to projects' check is done. There are {} inconsistency found.".format(count))
+
+
+if __name__ == '__main__':
+    check_consistency_of_users_to_projects()
```
Comment (sloria) [inline scripts/consistency/consistency_of_users_to_projects.py]: Use the `add_file_logger` utility so that the logs are written to a file.

**Comment by sloria** [comment]
Just a minor change and this is good to merge.

File: scripts/consistency/consistency_of_users_to_projects.py (hunk 1 above)
Comment (chennan47) [inline scripts/consistency/consistency_of_users_to_projects.py]: addressed in c9880b1

**Pull request merged**